### PR TITLE
Add setup instructions for WSL (Windows 10 environment)

### PR DESCRIPTION
**Pull request by ShuoZheLi**
1. Updated Readme.md to include a new section for installing WSL dependencies
2. Added a WSL_setup.sh for installing v86 dependencies

This solves the DOS file format issue in Windows environment.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -106,6 +106,15 @@ See `tools/docker/test-image/Dockerfile` for a full setup on Debian.
 - If you only want to embed v86 in a webpage you can use libv86.js. For
   usage, check out the [examples](examples/).
 
+For a full setup on Windows 10 WSL-2:
+
+- Install WSL-2 (Windows sub-linux system) from [here](https://docs.microsoft.com/en-us/windows/wsl/install)
+ - Install Windows sub-linux system ubuntu 20.04 from [here](https://www.microsoft.com/en-us/p/ubuntu-2004-lts/9n6svws3rx71?activetab=pivot:overviewtab)
+ - Install docsify-cli by using `npm i docsify-cli -g`
+ - Run `sudo dos2unix /usr/local/lib/node_modules/docsify-cli/bin/docsify`
```
Comment (copy) [inline Readme.md:114]: What is docsify used for?

Comment (ShuoZheLi) [reply Readme.md:114]: Hi sir:
        This is used to solve the problem of "/usr/bin/env: ‘node\r’: No such file or directory" when trying to build v86.

Comment (copy) [reply Readme.md:114]: That makes sense, but why is docsify needed in the first place?

**Comment by copy** [comment]
Thanks! I removed the docsify bit and merged the instructions with the existing Debian link above. Feel free to open another PR if you think anything is missing.

**Pull request merged**